ee-feature-pack/galleon-shared/src/main/resources/license/licenses.xml

```diff
@@ -4798,71 +4798,7 @@
-            <licenses>
-                <license>
-                    <name>Apache License 2.0</name>
-                    <url>http://www.apache.org/licenses/LICENSE-2.0</url>
-                    <distribution>repo</distribution>
-                </license>
-            </licenses>
-        </dependency>
-        <dependency>
-            <groupId>org.wildfly.core</groupId>
-            <artifactId>wildfly-cli</artifactId>
-            <licenses>
-                <license>
-                    <name>GNU Lesser General Public License v2.1 or later</name>
-                    <url>http://www.gnu.org/licenses/old-licenses/lgpl-2.1-standalone.html</url>
-                    <distribution>repo</distribution>
-                </license>
-                <license>
-                    <name>Apache License 2.0</name>
-                    <url>http://www.apache.org/licenses/LICENSE-2.0</url>
-                    <distribution>repo</distribution>
-                </license>
-            </licenses>
-        </dependency>
-        <dependency>
-            <groupId>org.wildfly.core</groupId>
-            <artifactId>wildfly-elytron-integration</artifactId>
-            <licenses>
-                <license>
-                    <name>GNU Lesser General Public License v2.1 or later</name>
-                    <url>http://www.gnu.org/licenses/old-licenses/lgpl-2.1-standalone.html</url>
-                    <distribution>repo</distribution>
-                </license>
-                <license>
-                    <name>Apache License 2.0</name>
-                    <url>http://www.apache.org/licenses/LICENSE-2.0</url>
-                    <distribution>repo</distribution>
-                </license>
-            </licenses>
-        </dependency>
-        <dependency>
-            <groupId>org.wildfly.core</groupId>
-            <artifactId>wildfly-event-logger</artifactId>
-            <licenses>
-                <license>
-                    <name>GNU Lesser General Public License v2.1 or later</name>
-                    <url>http://www.gnu.org/licenses/old-licenses/lgpl-2.1-standalone.html</url>
-                    <distribution>repo</distribution>
-                </license>
-                <license>
-                    <name>Apache License 2.0</name>
-                    <url>http://www.apache.org/licenses/LICENSE-2.0</url>
-                    <distribution>repo</distribution>
-                </license>
-            </licenses>
-        </dependency>
-        <dependency>
-            <groupId>org.wildfly.core</groupId>
-            <artifactId>wildfly-version</artifactId>
-            <licenses>
-                <license>
-                    <name>GNU Lesser General Public License v2.1 or later</name>
-                    <url>http://www.gnu.org/licenses/old-licenses/lgpl-2.1-standalone.html</url>
-                    <distribution>repo</distribution>
-                </license>
+            <licenses>`
```

Comment:
This ` passed unnoticed, it is fixed in https://github.com/wildfly/wildfly/pull/17218